jobbergate-cli/pyproject.toml

```diff
@@ -22,6 +22,16 @@ loguru = "^0.6.0"
 python-jose = "^3.3.0"
 pydantic = "^1.8.2"
 yarl = "^1.7.2"
+httpx = "^0.22.0"
+pendulum = "^2.1.2"
+py-buzz = "^3.0.0"
+snick = "^1.2"
+rich = "^11.2.0"
+typer = "^0.4.0"
+colorama = "^0.4.4"
+plummet = {extras = ["time-machine"], version = "^1.0.0"}
+auto-name-enum = "^1.1.0"
+pyperclip = "^1.8.2"
```

Comment:
Why so many new dependencies?

Reply:
Here's the breakdown:

- httpx: A replacement for requests. We are using this in most of our projects instead of requests
- pendulum: A datetime enhancement with improved parsing and formatting for user display
- py-buzz: Used as the base class for Abort. Provides the `require_conditions()`, `handle_errors()`, and `check_expressions()` helpers
- rich: Provides nicely formatted out put for CLI users including color and tables
- typer: A CLI building tool that adds automatic type checking and such for parameters.
- colorama: Just removed this. It was a leftover from some experimentation that I forgot to remove
- plummet: This adds testing utilities (`frozen_time`) to work with pendulum. I've moved this into the dev dependencies as it's only used for tests
- auto-name-enum: A very tiny package (it has no dependencies) that is useful for creating enums where the values match the names (this is one of my projects I have used a lot)
- pyperclip: This allows us to copy tokens to the clipboard for the `show-token` command. It's to enhance ease-of-use.